apps/aechannel/src/aesc_codec.hrl

```diff
@@ -64,8 +64,9 @@
 -define(CHANNEL_CLOSED, channel_closed).
 
 %% Error codes
--define(ERR_VALIDATION, 1).
--define(ERR_CONFLICT  , 2).
--define(ERR_TIMEOUT   , 3).
--define(ERR_ABORT     , 4).
+-define(ERR_VALIDATION        , 1).
+-define(ERR_CONFLICT          , 2).
+-define(ERR_TIMEOUT           , 3).
+-define(ERR_ABORT             , 4).
+-define(ERR_ONCHAIN_REJECTED  , 5).
```

Comment:
This should have a corresponding clause in `aesc_fsm:error_code_to_msg/1` (line 363)